test/snapshot.ts

```diff
@@ -117,23 +120,29 @@ describe('Snapshot', () => {
     });
 
     describe('with Subscription parent', () => {
+      let pubsub: PubSub;
+      let subscription: Subscription;
+      before(() => {
+        pubsub = new PubSub(PUBSUB);
+        subscription = pubsub.subscription('test');
+      });
       it('should include the create method', done => {
-        SUBSCRIPTION.createSnapshot = (name, callback) => {
-          assert.strictEqual(name, SNAPSHOT_NAME);
-          callback();  // The done function
-        };
+        sandbox.stub(subscription, 'createSnapshot')
+            .callsFake((name: string) => {
+              assert.strictEqual(name, FULL_SNAPSHOT_NAME);
+              done();
+            });
 
-        const snapshot = new Snapshot(SUBSCRIPTION, SNAPSHOT_NAME);
+        const snapshot = new Snapshot(subscription, SNAPSHOT_NAME);
         snapshot.create(done);
```

Comment:
Should probably pass `assert.ifError` instead of `done` since we're ignoring the callback inside of the stub.